tools/reviewer/web/src/app/dashboard/reviews-panel/reviews-panel.component.ts

```diff
@@ -39,23 +35,34 @@ export class ReviewsPanelComponent implements OnInit {
         // and create Diff from proto and categorize
         // into a specific list.
         for (const diff of diffs) {
-          for (const reviewer of diff.getReviewerList()) {
-            if (reviewer.getEmail() === this.authService.userEmail) {
-              if (reviewer.getNeedsattention()) {
-                // Need attention of user
-                this.diffGroups[Lists.NeedAttention].push(diff.toObject());
-              } else {
-                // User is cc'ed on this
-                this.diffGroups[Lists.CcedOn].push(diff.toObject());
-              }
-            } else if (reviewer.getEmail() === this.authService.userEmail) {
-              if (diff.getStatus() === Diff.Status.REVIEW_NOT_STARTED) {
+          diff.getDescription();
+          ///
+          const you: string = this.authService.userEmail;
+          // needAttentionOfList = author and all reviewers,
+          // where attention is requested
+          const needAttentionOfList: string[] = diff.getReviewerList()
+            .filter(reviewer => reviewer.getNeedsattention())
+            .map(reviewer => reviewer.getEmail());
+          if (diff.getAuthor().getNeedsattention()) {
+            needAttentionOfList.concat(diff.getAuthor().getEmail());
+          }
+
+          if (needAttentionOfList.includes(you)) {
+            // Need attention of user
+            this.diffGroups[Lists.NeedAttention].push(diff);
+          } else if (diff.getCcList().includes(you)) {
+            // User is cc'ed on this
+            this.diffGroups[Lists.CcedOn].push(diff);
+          } else if (diff.getAuthor().getEmail() === you) {
+            switch (diff.getStatus()) {
+              case Diff.Status.REVIEW_NOT_STARTED:
                 // Draft Review
-                this.diffGroups[Lists.NeedAttention].push(diff.toObject());
-              } else if (diff.getStatus() === Diff.Status.SUBMITTED) {
+                this.diffGroups[Lists.NeedAttention].push(diff);
+                break;
+              case Diff.Status.SUBMITTED:
                 // Submitted Review
-                this.diffGroups[Lists.CcedOn].push(diff.toObject());
-              }
+                this.diffGroups[Lists.CcedOn].push(diff);
```

Comment:
Also, not your change, but:
* This one should go to `SubmittedReviews`
* `SubmittedReviews` should be first, so if it's already submitted, it should not go into the other lists.